core/test/services/config.spec.js

```diff
@@ -124,4 +126,55 @@ describe('Config', () => {
       assert.equal(res.message, 'rejected', 'return error does not equal');
     });
   });
+
+  describe('User settings', () => {
+    before(() => {
+      global['localStorage'] = {
+        getItem: sinon.stub(),
+        setItem: sinon.stub()
+      };
+    });
+    afterEach(() => {
+      sinon.restore();
+      sinon.reset();
+    });
+    const key = 'myStorageKey';
+    const userSettingsObj = {
+      userSettings: {
+        userSettingGroups: {}
```

Comment:
to be consistent I would suggest to rename it to `userSettingsGroups`, we use everywhere the plural: userSettings